### DISCUSSION TITLE
Stable Diffusion 3 API Now Available

### DISCUSSION THREAD

**Discussion by zhaoqi571436204** [lllyasviel/Fooocus#2773] [General]
https://stability.ai/news/stable-diffusion-3-api

**Comment by Rogala** [comment]
[Stable Diffusion 3 First Impressions and Stable Assistant](https://www.youtube.com/watch?v=Yv8Sp1hvKNU)

**Comment by IPv6** [comment]
There is no ControlNets, no inpaint/outpaint, no LCM, no Loras... with altered architecture it is not so straightforward
Lets wait 4-6 months first :)

**Reply by Rogala** [reply]
It's still not a sure thing that there will be an update here, but there is hope

**Reply by mashb1t** [reply]
@Rogala more or less certain, see https://github.com/lllyasviel/Fooocus/discussions/2154#discussioncomment-8389770

**Comment by wisdombox44** [comment]
Be patient this is not Public Release yet. Here is what they said on twitter 

"@StabilityAI](https://twitter.com/StabilityAI)
While the model is available via API today as part of its initial launch, we are continuously working to improve the model in advance of its open release. You can anticipate seeing these improvements in the upcoming weeks."
I think they are testing their new paid API platfom as well. We may reach some time next month.

**Reply by Rogala** [reply]
I've already read it, thank you

[Stable Diffusion 3 API Now Available — Stability AI
[News](https://www.reddit.com/r/StableDiffusion/?f=flair_name%3A%22News%22)](https://www.reddit.com/r/StableDiffusion/comments/1c6awnl/stable_diffusion_3_api_now_available_stability_ai/?sort=top)

**Comment by Rogala** [comment]
[If you are very impatient, you can try SD3 via API ComfyUI Stable Diffusion 3](https://github.com/ZHO-ZHO-ZHO/ComfyUI-StableDiffusion3-API?tab=readme-ov-file)

[Stable Diffusion 3 - How to use it today! Easy Guide for ComfyUI](https://www.youtube.com/watch?v=SBkLY703Xs4)

**Comment by mashb1t** [comment]
"Stable Diffusion 3 Medium, our most advanced text-to-image is on its way! You will be able to download the weights on Hugging Face from Wednesday 12th June."

**Reply by mzisa** [reply]
Thank you! Where do they publish it? :)

**Comment by Rogala** [comment]
[I'm not a programmer, would like to understand how long and how realistic it is to hook them up to Fooocus?](https://stability.ai/stablediffusion3)

**Reply by mashb1t** [reply]
I've already started to update all external dependencies in https://github.com/lllyasviel/Fooocus/pull/3084 to make it easier to integrate SD3 on launch, but am currently facing a challenge. @lllyasviel support would be highly appreciated.

**Reply by Rogala** [reply]
Another question about the hardware, how do the minimum requirements for CPU, GPU, RAM change, any hints, I have a very weak PC? 

Maybe I'm getting excited for nothing.

**Reply by xhoxye** [reply]
Looking forward to @lllyasviel's support

**Reply by mashb1t** [reply]
@Rogala as SD3 Medium is announced to have 2b parameters and SDXL is a ~2.3b~ 3.5b model, they should be pretty similar (at least model only, not accounting for external ControlNet etc.).

**Reply by xhoxye** [reply]
https://stability.ai/news/stable-diffusion-sdxl-1-announcement
SDXL 1.0 has one of the largest parameter counts of any open access image model, boasting a 3.5B parameter base model and a 6.6B parameter model ensemble pipeline (the final output is created by running on two models and aggregating the results).
3.5B-8G

**Reply by mashb1t** [reply]
@xhoxye thanks for the clarification, 3,5b is correct!

**Comment by Rogala** [comment]
[Civitai Temporary Stable Diffusion 3 Ban](https://civitai.com/articles/5732/temporary-stable-diffusion-3-ban)
